uwblueprint/feeding-canadian-kids
# Benymng/setup google geoncoding api

## Notion Ticket Link

[Setup Google Geoencoding API](https://www.notion.so/uwblueprintexecs/Setup-Google-Geoncoding-API-162d49b825bb4a189a3993f5b34763c2?pvs=4)



## Implementation Description
* Use the google API to get the longitude and latitude of the onboarding address



## Steps To Test
1. Create an onboarding request and see if the latitude and latitude associated with the user are correct



## What Should Reviewers Focus On?
*


## Checklist
